app/javascript/packs/application.js

```diff
@@ -14,9 +14,26 @@ Rails.start();
 Turbolinks.start();
 ActiveStorage.start();
 
+export const scroll_bottom = () => {
+  if ($(".chatroom-messages").length > 0) {
+    $(".chatroom-messages").scrollTop($(".chatroom-message-container")[0].scrollHeight);
+  }
+}
+
+const submit_message = () => { 
+  $("#message_body").on("keydown", (e) => {
```

Comment:
Why do you need this listener?

Reply:
Since I've removed the button from the display, hitting on the `Enter` key on the input field is the only way to send the message atm, so we're listening to that event to submit the message.